wiki/dfg_nodes.md

```diff
@@ -0,0 +1,53 @@
+# Understanding Data Flow Graph (DFG) Node Labels in Flowr
+
+## Overview
+The Data Flow Graph (DFG) in Flowr represents the dependencies between different code components. Each node in the DFG contains specific labels that provide information about the operation being performed, the code span, and its relationship with other nodes.
+
+This document clarifies the meaning of these labels and how to interpret them.
+
+## Node Label Structure
+Each node in the DFG follows this general format:
+
+```
+[NodeType] (NodeID) LineRange ParentNodes
+```
+
+### **Breakdown of Components:**
+- **NodeType:** Indicates the type of operation. Example: `[RBinaryOp]` for a binary operation.
```

Comment:
This is the syntactical type of the operation and controlled by the [syntactic type](https://github.com/flowr-analysis/flowr/wiki/Normalized-AST). It is the shape of the vertex that indicates its type in the dataflow graph. 
The ["Example: Assigning with a String"](https://github.com/flowr-analysis/flowr/wiki/Dataflow-Graph#4-variable-definition-vertex) section of the DFG wiki presents a nice example of when they may differ.